packages/@webex/plugin-cc/src/services/core/WebSocket/connection-service.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable */
+import { Signal } from '../Signal';
```

Comment:
This file has the exact same reconnection logic taken from agent desktop. Checked and it works, no major changes have been made, we have only replaced `aqm-notifs` with `WebSocketManager`.